### PR TITLE
XP Changes

### PR DESCRIPTION
**Pull request by daanyx**
Mudanças no campo de caça:
https://www.tibia.com/forum/?action=announcement&announcementid=98&boardid=89516

Conferido em:
https://tibia.fandom.com/
https://www.tibiawiki.com.br/wiki/Home

### CODE REVIEW

**Comment by dex-89** [comment]
Thank you for your contribution.

File: data/monster/elementals/knowledge_elemental.lua [hunk 1]
```diff
@@ -2,7 +2,7 @@ local mType = Game.createMonsterType("Knowledge Elemental")
 local monster = {}
 
 monster.description = "a knowledge elemental"
-monster.experience = 10710
+monster.experience = 10603
```
Comment (dex-89) [inline data/monster/elementals/knowledge_elemental.lua:5]: 11781 (tibia.com)

File: data/monster/magicals/brain_squid.lua [hunk 2]
```diff
@@ -2,7 +2,7 @@ local mType = Game.createMonsterType("Brain Squid")
 local monster = {}
 
 monster.description = "a brain squid"
-monster.experience = 17850
+monster.experience = 17672
```
Comment (dex-89) [inline data/monster/magicals/brain_squid.lua:5]: 19635 (tibia.com)

File: data/monster/magicals/energetic_book.lua [hunk 3]
```diff
@@ -2,7 +2,7 @@ local mType = Game.createMonsterType("Energetic Book")
 local monster = {}
 
 monster.description = "an energetic book"
-monster.experience = 12155
+monster.experience = 12034
```
Comment (dex-89) [inline data/monster/magicals/energetic_book.lua:5]: 13371 (tibia.com)

File: data/monster/magicals/energuardian_of_tales.lua [hunk 4]
```diff
@@ -2,7 +2,7 @@ local mType = Game.createMonsterType("Energuardian of Tales")
 local monster = {}
 
 monster.description = "an energuardian of tales"
-monster.experience = 11475
+monster.experience = 11361
```
Comment (dex-89) [inline data/monster/magicals/energuardian_of_tales.lua:5]: 12623 (tibia.com)

**Review by dex-89 — CHANGES_REQUESTED** [review]
Fix experience with tibia.com

File: data/monster/elementals/knowledge_elemental.lua (hunk 1 above)
Comment (daanyx) [reply data/monster/elementals/knowledge_elemental.lua:5]: https://www.tibia.com/library/?subtopic=creatures&race=knowledgeelemental

File: data/monster/magicals/brain_squid.lua (hunk 2 above)
Comment (daanyx) [reply data/monster/magicals/brain_squid.lua:5]: https://www.tibia.com/library/?subtopic=creatures&race=brainsquid

File: data/monster/magicals/energetic_book.lua (hunk 3 above)
Comment (daanyx) [reply data/monster/magicals/energetic_book.lua:5]: https://www.tibia.com/library/?subtopic=creatures&race=energeticbook

File: data/monster/magicals/energuardian_of_tales.lua (hunk 4 above)
Comment (daanyx) [reply data/monster/magicals/energuardian_of_tales.lua:5]: https://www.tibia.com/library/?subtopic=creatures&race=energuardianoftales

File: data/monster/elementals/knowledge_elemental.lua (hunk 1 above)
Comment (dex-89) [reply data/monster/elementals/knowledge_elemental.lua:5]: It was updated. Library (tibia.com) is not good base
image

Comment (daanyx) [reply data/monster/elementals/knowledge_elemental.lua:5]: Veja o histórico das mudanças.
https://tibia.fandom.com/wiki/Knowledge_Elemental

Comment (dex-89) [reply data/monster/elementals/knowledge_elemental.lua:5]: Which is current? Idk...
image

Comment (daanyx) [reply data/monster/elementals/knowledge_elemental.lua:5]: Prefiro acreditar no site (tibia.com) 
Captura de tela 2020-12-21 150650

Brain Squid 19635-1963= (17672)
https://www.tibia.com/library/?subtopic=creatures&race=brainsquid

Energetic Book 13371-1337= (12034)
https://www.tibia.com/library/?subtopic=creatures&race=energeticbook

Energuardian of Tales 12623-1262= (11361)
https://www.tibia.com/library/?subtopic=creatures&race=energuardianoftales

Knowledge Elemental 11781-1178= (10603)
https://www.tibia.com/library/?subtopic=creatures&race=knowledgeelemental

The XP of the Secret Library energy section creatures has been lowered by 10%.
https://tibia.fandom.com/wiki/Updates/12.50.10135

Comment (dex-89) [reply data/monster/elementals/knowledge_elemental.lua:5]: I think other contributors should comment

**Pull request closed without merging**